spec/fixtures/bookmark.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 28">
```

Comment:
Where is this coming from?  Do we have license to include it with Blacklight?

Reply:
Shoot meant to grab the blacklight icons one. This is the one from geoblacklight.